spec/rspec/core/runner_spec.rb

```diff
@@ -93,29 +96,29 @@ module RSpec::Core
     end
 
     describe "interrupt handling" do
-      before { allow(Runner).to receive(:exit!) }
+      before { allow(subject).to receive(:exit!) }
 
       it 'prints a message the first time, then exits the second time' do
         expect {
-          Runner.handle_interrupt
+          subject.send(:handle_interrupt)
```

Comment:
As above suggestions, names are important and we don't recommend using `subject` explicitly.

Reply:
Okay great.